hw/ip/otbn/util/BUILD

```diff
@@ -2,6 +2,40 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
+load("@rules_python//python:defs.bzl", "py_binary")
+load("@ot_python_deps//:requirements.bzl", "requirement")
+
 package(default_visibility = ["//visibility:public"])
 
 exports_files(glob(["**"]))
+
+py_binary(
+    name = "otbn_as",
+    srcs = ["otbn_as.py"],
+    deps = [
+        "//hw/ip/otbn/util/shared:bit_ranges",
+        "//hw/ip/otbn/util/shared:encoding",
+        "//hw/ip/otbn/util/shared:insn_yaml",
+        "//hw/ip/otbn/util/shared:operand",
+        "//hw/ip/otbn/util/shared:toolchain",
```

Comment:
Wouldn't it make more sense to package up all the stuff in the `shared` directory into a single `py_library` and then depend on that? I don't think anyone will mind e.g. a couple of stray re-links if we change the `operand` code. That way, you don't have to list files in more than one place.

I'd also suggest using a `glob(...)` rule to get the Python files in that directory. I'm not sure I can see an advantage of forcing oneself to list the files separately.